spec/services/cancel_interview_spec.rb

```diff
@@ -45,5 +45,11 @@
       ])
       expect(associated_audit.audited_changes['cancellation_reason']).to eq([nil, 'There is a global pandemic going on'])
     end
+
+    it 'sends an email to the candidate', sidekiq: true do
+      CancelInterview.new(service_params).save!
+
+      expect(ActionMailer::Base.deliveries.first['rails-mail-template'].value).to eq('interview_cancelled')
+    end
```

Comment:
Can we merge this with the previous spec. 

Reason being the time it takes to create an application choice. As they are both mandatory when cancelling an interview, we can just test them as part of the same spec.